### DISCUSSION TITLE
Simplify search qualifiers

### DISCUSSION THREAD

**Discussion by ns-wkso** [community/community#10383] [Other Feature Feedback, Questions, & Ideas]
Can we make the qualifiers simpler?   Like we support language: and repo: and path: et al, but how about l: r: and p: also?   Sourcegraph and other code search solutions do this.

**Comment by colinwm** [comment]
We do have an undocumented `lang:` shorthand for `language:`. But I think four characters should be OK for a qualifier, right? By making them much shorter the query language becomes more overloaded and cryptic, so I'm hesitant to do it. 

But if lots of people agree, maybe we can reconsider.

**Comment by ns-wkso** [comment]
I was told engineers are lazy, so the shorter the better but I think lang is good.  Are there other undocumented ones?  Would they be documented at some point? I guess now we have to do path: to find specific file name.  Is there a file: or f: or something more intuitive when searching for file name?
